### PR TITLE
Rework the QGIS Server Apache section

### PR DESCRIPTION
**Pull request by elemoine**
### Description

Rework the QGIS Server section that describes how to set up QGIS Server with Apache.

- [x] Backport to LTR documentation is required

### Minimal requirements for merging *(for maintainers)*

- [ ] The description of this PR is coherent with the manual and does not provide wrong information.
- [ ] This PR passes the checks.

### CODE REVIEW

**Comment by elemoine** [comment]
I think this is a good move for the QGIS Server documentation. Anyone up for a quick review? Thanks.

**Comment by DelazJ** [comment]
Hi @elemoine 
I'm not skilled to review but from what i can read, I wonder why you remove the backport to 3.4 checkbox. Is there anything that made this process incompatible with 3.4? Thanks.

**Comment by elemoine** [comment]
> Hi @elemoine
> I'm not skilled to review but from what i can read, I wonder why you remove the backport to 3.4 checkbox.

By mistake. I added it back.



> .Is there anything that made this process incompatible with 3.4? Thanks.

No there isn't. So I guess the checkbox should be checked, right?

File: source/docs/user_manual/working_with_ogc/server/getting_started.rst [hunk 1]
```diff
@@ -31,78 +31,76 @@ Please refer to that section.
...
-Enable the rewrite module to pass HTTP BASIC auth headers:
+Install Apache and ``mod_fcgid``:
 
 .. code-block:: bash
 
- sudo a2enmod rewrite
- cat /etc/apache2/conf-available/qgis-server-port.conf
- Listen 80
- sudo a2enconf qgis-server-port
+ sudo apt-get apache2 libapache2-mod-fcgid
```
Comment (Gustry) [inline source/docs/user_manual/working_with_ogc/server/getting_started.rst]:
```suggestion
 sudo apt install apache2 libapache2-mod-fcgid
```

File: source/docs/user_manual/working_with_ogc/server/getting_started.rst [hunk 2]
```diff
@@ -31,78 +31,76 @@ Please refer to that section.
...
+    CustomLog ${APACHE_LOG_DIR}/access.log combined
+
    </VirtualHost>
 
+.. note::
 
-Moreover, you can use some :ref:`qgis-server-envvar` to configure QGIS Server.
-With Apache as HTTP Server, you have to use ``FcgidInitialEnv`` to define these
-variables as shown below:
+   On Debian systems a default ``VirtualHost`` is available in
```
Comment (Gustry) [inline source/docs/user_manual/working_with_ogc/server/getting_started.rst]:
```suggestion
   On Debian systems, a default ``VirtualHost`` is available in
```

File: source/docs/user_manual/working_with_ogc/server/getting_started.rst (hunk 1 above)
Comment (elemoine) [reply source/docs/user_manual/working_with_ogc/server/getting_started.rst]: Thanks! I kept `apt-get` as it is used elsewhere in this page.

File: source/docs/user_manual/working_with_ogc/server/getting_started.rst (hunk 2 above)
Comment (elemoine) [reply source/docs/user_manual/working_with_ogc/server/getting_started.rst]: Fixed.

**Comment by elemoine** [comment]
@Gustry thank you for your suggestions. I amended my commit.

File: source/docs/user_manual/working_with_ogc/server/getting_started.rst (hunk 1 above)
Comment (Gustry) [reply source/docs/user_manual/working_with_ogc/server/getting_started.rst]: `apt install` is more friendly I think. Should we switch to this new syntax?

Comment (elemoine) [reply source/docs/user_manual/working_with_ogc/server/getting_started.rst]: I really don't care personally :) `apt-get` is used everywhere else in the QGIS Documentation, and changing to `apt` from `apt-get` means editing multiple rst files. I can do that but I think it'd be better to do this with a separate PR.

Comment (Gustry) [reply source/docs/user_manual/working_with_ogc/server/getting_started.rst]: Sure, I didn't ask you to change everywhere. It was just a question ;-)

**Comment by DelazJ** [comment]
Thanks @Gustry for the review. Good to merge?

**Pull request merged**